src/extension/app/store/app.js

```diff
@@ -145,6 +163,10 @@ export class AppStore {
       config: this.siteStore.toJSON(),
       location: this.location,
     });
+
+    this.sidekick.addEventListener('statusfetched', () => {
+      this.showView();
```

Comment:
Why do we need to wait for the status to show the view?

Reply:
Took it from [you](https://github.com/adobe/helix-sidekick-extension/blob/main/src/extension/module.js#L3097) ;)

Let me try without, I think you may be right.

Reply:
Looks like we need the status for the [webpath](https://github.com/adobe/aem-sidekick/pull/84/files#diff-2232b3a8d76d8fdfb8c5d8f3d8fe25a9fc392b09dce95767f015a1c382061104R690)

Reply:
Hmm, yeah, I guess that was when custom and built-in views started colliding... and I ended up intertwining the two more than was necessary. We do need to know if we should load a custom viewer before switching to preview or live from other environments such as edit, because the URL will be different. That is why the `webPath` is needed there. However, in the case of this built-in view, the URL will always be the same, so we don't need to wait for the status and can simply move along with the  `window.location.pathname`. Would probably speed things up considerably.